quine/src/main/scala/com/thatdot/quine/app/ingest/KafkaSrcDef.scala

```diff
@@ -46,6 +46,7 @@ object KafkaSrcDef {
     bootstrapServers: String,
     groupId: String,
     autoOffsetReset: KafkaAutoOffsetReset,
+    kafkaProperties: Option[KafkaIngest.KafkaProperties],
```

Comment:
Seems this could also just be `KafkaProperties`, with the default value of empty Map. I assume the `Option` is for how endpoints4s deals with default values, or displays that in swagger?

Reply:
Defaulting it all that back at the `IngestRoutes.scala` seems to work fine as well.
```scala
@docs(
    "Map of Kakfa client properties. See <https://docs.confluent.io/platform/current/installation/configuration/consumer-configs.html#ak-consumer-configurations-for-cp>"
  )
  kafkaProperties: KafkaIngest.KafkaProperties = Map.empty[String, String],
  ```